src/PathTracedVolume.ts

```diff
@@ -356,16 +337,80 @@ export default class PathTracedVolume implements VolumeRenderImpl {
     this.sampleCounter = 0;
   }
 
-  public setVisible(_isVisible: boolean): void {
-    // this.visible = isVisible;
-  }
+  /**
+   * If the new settings have changed, applies all changes to this volume renderer.
+   * @param newSettings
+   * @returns
+   */
+  public updateSettings(newSettings: VolumeRenderSettings, _dirtyFlags?: number | SettingsFlags): void {
+    if (_dirtyFlags === undefined) {
+      _dirtyFlags = SettingsFlags.ALL;
+    }
+    if (_dirtyFlags === SettingsFlags.NONE) {
+      return;
+    }
+    this.settings = newSettings;
+
+    // Update resolution
+    if (_dirtyFlags & SettingsFlags.RESOLUTION_AND_SAMPLING) {
```

Comment:
very minor/optional: resolution is also sampling so I suppose this could just be named SAMPLING!